vm/systemSmartContracts/staking.go

```diff
@@ -231,21 +231,26 @@ func (r *stakingSC) calculateStakeAfterBleed(startRound uint64, endRound uint64,
 
 func (r *stakingSC) changeValidatorKey(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 	if !bytes.Equal(args.CallerAddr, r.stakeAccessAddr) {
-		log.Debug("stake function not allowed to be called by", "address", args.CallerAddr)
+		log.Debug("changeValidatorKey function not allowed to be called by", "address", args.CallerAddr)
+
```

Comment:
delete empty line - no need for log.debug

Reply:
done.